Gems/WhiteBox/Code/Source/SubComponentModes/EditorWhiteBoxTransformMode.h

```diff
@@ -8,12 +8,25 @@
 
 #pragma once
 
+#include "EditorWhiteBoxComponentMode.h"
 #include "EditorWhiteBoxComponentModeTypes.h"
+#include "Viewport/WhiteBoxManipulatorViews.h"
+
+#include <AzCore/Math/Quaternion.h>
+#include <AzCore/std/containers/span.h>
+#include <AzCore/std/containers/variant.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/std/optional.h>
+#include <AzCore/std/smart_ptr/shared_ptr.h>
+#include <AzCore/std/smart_ptr/unique_ptr.h>
+#include <WhiteBox/WhiteBoxToolApi.h>
```

Comment:
question: It does seem like some of these includes are either not required or could be moved to the .cpp file, might be worth doing a quick check. Thanks!